__tests__/endpoint.test.js

```diff
@@ -39,4 +39,55 @@ describe('GET /api/topics', () => {
             expect(body.msg).toBe('404: Page Not Found.')
         } )
     })
+})
+
+describe('PATCH /api/articles/:article_id', () => {
+    test('status:200, responds with votes increased by amount added', () => {
+        const articleUpdates = {
+            inc_votes: 3
+        };
+        return request(app)
+        .patch('/api/articles/1')
+        .send(articleUpdates)
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toEqual({ newVotesTotal: 103 })
+        })
+    })
+    test('Works with negative numbers', () => {
+        const articleUpdates = {
+            inc_votes: -50
+        };
+        return request(app)
+        .patch('/api/articles/1')
+        .send(articleUpdates)
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toEqual({ newVotesTotal: 50})
+        })
+    })
+    test('If the number would go below 0, it remains at 0', () => {
+        const articleUpdates = {
+            inc_votes: -500
+        };
+        return request(app)
+        .patch('/api/articles/1')
+        .send(articleUpdates)
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toEqual({ newVotesTotal: 0})
+        })
+    } )
```

Comment:
Nice idea, but if you were using this on a website would you want people to be able to give an article negative votes or just stop stop the voting at zero?